tests/run_tests.sh

```diff
@@ -383,6 +384,26 @@ function run_bsl_tests()
 
 setup_environment
 
+for arg in "$@"; do
+    if [[ "$arg" == "--enable-debug" ]]; then
+        if [[ -z $SONIC_MGMT_DEBUG_PORT ]]; then
+            echo "*********************************[WARNING]*********************************"
+            echo "This container was not setup with --enable-debug option. Please re-setup this container with --enable-debug option"
+            echo "Please re-run without '--enable-debug' option to continue."
+            echo "*********************************[WARNING]*********************************"
+            exit 1
+        fi
+
+        if [[ "$arg" != "${@: -1}" ]]; then
+            echo "Please put '--enable-debug' as the last option of your './run_tests.sh' command to avoid conflicts with pytest and run_tests options"
```

Comment:
Just a thinking, not sure if it can simplifies the process:
Can we make this automatically.
Like if we pass --enable-debug when setting up the container,
we set an environment variable(or a flag file) to mark we enables the debug mode.
And when we run run_test, it reads the env(or flag file), enables debug mode automatically.
But not sure whether there's scenario that we want to explicitly disable debug mode when run_test

Reply:
yes this can be done, however the user won't be able to run without the debug mode.

The thing is if we do enable debug by default, debugpy won't trigger the test case until there is a client connection. So the users always have to connect their vscode client to run a test whether or not they want debugging.